src/LuaSystemBody.cpp

```diff
@@ -276,6 +276,28 @@ static int l_sbody_attr_apoapsis(lua_State *l)
 	return 1;
 }
 
+
+/*
+ * Attribute: orbitPeriod
+ *
+ * The orbit of the body, around its parent, in days
+ *
+ * Availability:
+ *
+ *   201708
+ *
+ * Status:
+ *
+ *   experimental
+ */
+static int l_sbody_attr_orbital_period(lua_State *l)
+{
+	SystemBody *sbody = LuaObject<SystemBody>::CheckFromLua(1);
+	lua_pushnumber(l, sbody->GetOrbit().Period() / float(60*60*24));
```

Comment:
Eh, that's an elegant way to do the cast, I'd forgotten about this syntaxic possibility :-)